pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -133,13 +133,15 @@ const (
 
 // predicateOrdering is the ordering of predicate execution.
 var predicateOrdering = []string{
+	GeneralPred,
```

Comment:
What is the principle for reordering plugins that are not in default filtter pluginset, e.g. GeenralPred, CheckNodeLabelPresencePred?

BTW: since it is legacy registry, do we need to reorder them?